doc/classes/TileMapLayer.xml

```diff
@@ -22,6 +22,14 @@
 				[b]Note:[/b] If the properties of [param tile_data] object should change over time, use [method notify_runtime_tile_data_update] to notify the [TileMapLayer] it needs an update.
 			</description>
 		</method>
+		<method name="_update_cells" qualifiers="virtual">
+			<return type="void" />
+			<param index="0" name="coords" type="Vector2i[]" />
+			<description>
+				Called with the cells of this [TileMapLayer] that are queued for an internal update. Such update might be a caused by a change in the [TileSet] (causes all cells to be queued for update) or from a call to [method set_cell]. [param coords] contains the position of all the modified cells.
```

Comment:
Simplifying wording a bit. It conflicts a bit with @groud 's previous suggestion.

From the outside perspective there's more ways to modify cells, not just `set_cell`.

I believe the order of the cells can be pretty important (assuming the statement is accurate).

```suggestion
				Called when this [TileMapLayer]'s cells need an internal update. This update may be caused from individual cells being modified or by a change in the [member tile_set] (causing all cells to be queued for an update). [param coords] contains the coordinates of all modified cells, roughly in the order they were modified. See also [method update_internals].
```

Reply:
Yeah this new wording is better while keeping the meaning. We should probably mention that implementing this function might have a performance impact too though, as it requires creating an array of coordinates.

Reply:
I don't know. You can reasonably expect a performance impact when overriding virtual functions. If it's still considerably faster than say, a manual check every process frame, I don't think it's worth bringing it up.

Reply:
> I don't know. You can reasonably expect a performance impact when overriding virtual functions. If it's still considerably faster than say, a manual check every process frame, I don't think it's worth bringing it up.

No, it is worth. A TileMap can handle hundreds of thousands of tiles. Allocating an filling such an array is impactful on performance, so it should be mentioned IMO.